### PR TITLE
Handle online events with URL rather than address

### PR DESCRIPTION
**Pull request by 0916dhkim**
#### Changes
- If `@location` is a URL, do not format it like a venue.
- Add test cases for `Event.slack_where` method.
- Add `engines` property to `package.json`.

### CODE REVIEW

File: event.coffee [hunk 1]
```diff
@@ -88,6 +88,7 @@ class Event extends Model
     "<!date^#{@starts_at_stamp}^{date_pretty} at {time}|#{@starts_at}>"
 
   @getter 'slack_where', ->
+    return "<#{@location}>" if URL_REGEX.test(@location)
```
Comment (kieran) [inline event.coffee]: might be nice to add comments:
```coffee
  @getter 'slack_where', ->
    # does it look like a URL?
    return "<#{@location}>" if URL_REGEX.test @location

    # probably an address, let's search for it
    "<#{@map_url}|#{@venue}>" if @map_url
```

might also be fun to try to handle addresses / plain text differently
```coffee
  @getter 'slack_where', ->
    # does it look like a URL?
    return "<#{@location}>" if URL_REGEX.test @location

    # does it look like an address? (is there a number?)
    return "<#{@map_url}|#{@venue}>" if /\d/.test @location

    # I'm out of ideas, just print the text
    @location
```
(would need a test to see if this works)

File: package.json [hunk 2]
```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "npx coffee index.coffee",
     "dev": "npx nodemon index.coffee",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "npx coffee event.test.coffee"
```
Comment (kieran) [inline package.json:9]: <3

**Review by kieran — COMMENTED** [review]
looking great! love the tests!

lmk what you think re: addresses / plain text

File: event.coffee (hunk 1 above)
Comment (0916dhkim) [reply event.coffee]: That's a good point. Added a fallback & `/\d/` checking :+1:

**Review by kieran — APPROVED** [review]
approve

Thanks for doing this! Deploying asap :-)

**Pull request merged**